app/views/hosts/_form.html.erb

```diff
@@ -22,6 +23,10 @@
       <% end %>
       <li><a href="#params" id='params-tab' data-url='<%= current_parameters_hosts_path %>' data-url2='<%= puppetclass_parameters_hosts_path %>' data-toggle="tab"><%= _('Parameters') %></a></li>
       <li><a href="#info" data-toggle="tab"><%= _('Additional Information') %></a></li>
+
+      <% @tabs.keys.each do |tab| %>
```

Comment:
I'd expect params, info, etc to be aspects as well. this adds clutter instead of removing it...